src/components/Tasks/TasksTable.js

```diff
@@ -49,17 +53,23 @@ const TaskTable = React.memo(function TaskTable({
             </div>
           </div>
           {!compactMode && !onlyTable && (
-            <TasksFilters
-              dateRangeFilter={dateRangeFilter}
-              onDateRangeChange={onDateRangeChange}
-              phaseFilter={phaseFilter}
-              onPhaseChange={onPhaseChange}
-              phaseTypes={phaseTypes}
-            />
+            <React.Fragment>
+              <TasksFilters
+                dateRangeFilter={dateRangeFilter}
+                onDateRangeChange={onDateRangeChange}
+                phaseFilter={phaseFilter}
+                onPhaseChange={onPhaseChange}
+                phaseTypes={phaseTypes}
+              />
+            </React.Fragment>
           )}
         </>
       }
-      fields={['Action', 'Dispute', 'Assigned to juror', 'Status', 'Due date']}
+      fields={
+        emptyFilterResults
+          ? []
+          : ['Action', 'Dispute', 'Assigned to juror', 'Status', 'Due date']
```

Comment:
What do you think of making this . a const variable at the component level ?

Reply:
i think that since this is just the columns names is easier to find there if we need to change something instead of looking for the variable and change there,
[Aragon apps also do that](https://github.com/aragon/aragon-apps/blob/f402373aae2237ac4ce91779a6504e3f4eb0452f/apps/finance/app/src/components/Transfers.js#L257)